cassovary-core/src/test/scala/com/twitter/cassovary/util/io/ListOfEdgesGraphReaderSpec.scala

```diff
@@ -53,6 +56,37 @@ class ListOfEdgesGraphReaderSpec extends WordSpec with GraphBehaviours[Node] wit
       idReader = _.toLong).toSharedArrayBasedDirectedGraph()
   }
 
+  trait GraphWithStringIdsWithSortReader {
```

Comment:
That's right. I am aware of it it but wanted to make sure the approach is correct. I will keep separate tests but reduce traits.